.travis.yml

```diff
@@ -8,9 +8,9 @@ services:
   - postgresql
 
 addons:
+    postgresql: "9.2"
```

Comment:
Is your spacing incorrect? Travis is failing :(

Reply:
Fixed!